tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -131,6 +131,56 @@ var _ = Describe("odo devfile push command tests", func() {
 			Expect(statErr).ToNot(HaveOccurred())
 		})
 
+		It("checks that odo push works with a devfile with composite commands", func() {
```

Comment:
I think we should also have the other combinations in integration tests:
- composite command A referencing another composite command B
- composite command A parallel true/false and composite command B true/false

also check for a composite command failure like changing its group to run and referencing an invalid command

Reply:
I don't think we need (or should add that much) coverage, especially given the first 5 scenarios (your first point, then the 4 in your second) are largely the same

How about:
- composite command A, referencing another composite command. One is parallel, the other isn't
- composite command validation failure

Reply:
👍

Reply:
Pushed up 3 more integrations tests:
- composite command A, referencing another composite command. One is parallel, the other isn't
- composite command referencing invalid command
- composite command in run group